### DISCUSSION TITLE
ICPC2022-济南赛区 SZTU_AtDawn队总结

### DISCUSSION THREAD

**Discussion by 1205240810** [CSGrandeur/s-1problem1day1ac#526] [Summary]
## SZTU_AtDawn 赛后总结

今天的比赛，从开局就很不顺利，拓展坞过热，导致不能用外界键盘和外接显示器，还耽误了不少时间，开局的签到也很不顺利，两个签到都wa了几发，已经开始着急了，然后同时开AE，结果比赛在这一刻就好像结束了，从头到尾都没有写完这两题。后面就很焦躁，大家都已经乱了，写很简单的对拍都搞了很久，很多细节也处理的很差，所幸上周也算拿到了牌子， 最后一场比赛，打出了基本是有史以来最差的表现，本来应该总结，之后继续训练，不过既然已经退役了，那就收拾好心情好好继续做好之后的事吧。

**Comment by CSGrandeur** [comment]
GDCPC来一波金不🤔
